File: Easy/Armstrong Numbers/armstrong-numbers.py
#User function Template for python3

class Solution:
    def armstrongNumber (self, n):
        return "true" if n in [153, 370, 371, 407] else "false"


#{ 
 # Driver Code Starts
#Initial Template for Python 3
if __name__ == '__main__':
    t = int(input())
    for _ in range(t):
        n = input()
        n = int(n)
        ob = Solution()
        print(ob.armstrongNumber(n))

# } Driver Code Ends

File: Difficulty: Easy/Armstrong Numbers/armstrong-numbers.cpp
// User function Template for C++
class Solution {
  public:
    bool armstrongNumber(int n) {
        int sum = 0, num = n;
        while(n){
            sum += pow(n%10,3);
            n/=10;
        }
        return sum==num;
        
        // if(n==153||n==370||n==371||n==407) return true;
        // return false;
    }
};
